charts/gateway/values.yaml

```diff
@@ -411,8 +411,8 @@ service:
     #   protocol: TCP
   # Additional Service annotations, the example below shows configuration for an internal load balancer on GCP
   # See the docs for more ==> https://kubernetes.io/docs/concepts/services-networking/service/
-  annotations: {}
-    # cloud.google.com/load-balancer-type: "Internal"
+  annotations:
+    cloud.google.com/load-balancer-type: "Internal"
```

Comment:
Uncommented during GKE testing. I forgot to comment this back.

Reply:
Resolved by https://github.com/CAAPIM/apim-charts/pull/193/commits/32daaae94b8d7f0ffc7f9d15b56c23e82e7e7ace